internal/transformer/transformresult.go

```diff
@@ -150,384 +146,389 @@ func transformReadBase(value interface{}, base string, lc logger.LoggingClient)
 	case float64:
 		value = valueFloat64
 	}
-	return value, err
+	return value, nil
 }
 
-func transformReadScale(value interface{}, scale string, lc logger.LoggingClient) (interface{}, error) {
+func transformReadScale(value interface{}, scale string) (interface{}, errors.EdgeX) {
 	switch v := value.(type) {
 	case uint8:
-		s, err := strconv.ParseFloat(scale, 64)
+		s, err := strconv.ParseUint(scale, 10, 8)
 		if err != nil {
-			lc.Error(fmt.Sprintf("the scale %s of PropertyValue cannot be parsed to %T: %v", scale, v, err))
-			return value, err
+			errMsg := fmt.Sprintf("the scalse value %s of PropertyValue cannot be parsed to %T", scale, v)
```

Comment:
Fix typo `scalse ` => `scale` here and below.

Reply:
fixed